recipes/mlpy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "3.5.0" %}
+{% set sha256 = "344fa75fbf9f76af72f6a346d5309613defc4d244bac13c218e509a51d68bf6a" %}
+#source file is named download, has to be unpacked manually, update build sh accordingly when updating version
+
+package:
+  name: mlpy
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/mlpy/files/mlpy%203.5.0/mlpy-{{ version }}.tar.gz/download
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+#  skip: True # [py27]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python >=2.6
+    - setuptools
+    - numpy =1.5
+    - scipy =0.11
+    - gsl =1.16
```

Comment:
Do these need to be pinned precisely? Can you use `>=` or at least `1.5.*`? It's easier to get tools to cooperate in an env if there is some leeway on what they require.

Reply:
This are the newest versions that worked for me, unless I can pin them >=x<=y I do not see another way

Reply:
Thx for the tip, wasn't aware of that, quite useful